### DISCUSSION TITLE
Plot G having the adjacency matrix

### DISCUSSION THREAD

**Discussion by leilaher** [networkx/networkx#4955] [Q&A]
Hello all,
How can I obtain a graph G having the adjacency matrix? Essentially the opposite of what the function "networkx.linalg.graphmatrix.adjacency_matrix" does.
My graph is a DAG, a Directed Acyclic Graph, with no contemporaneous links. I have the adjacency matrix and I want to graph the DAG.
Thanks,
Leila Hernandez

**Comment by dschult** [comment]
There are a number of ways to convert from data objects to a graph. Many are adjacency arrays or matrices. Others are edge lists.

https://networkx.org/documentation/stable/reference/convert.html

If it is in a data file, then take a look at the read/write options:

https://networkx.org/documentation/stable/reference/readwrite/index.html
